eprei/ft_transcendence
# E invite to play

Invite another user to play

The management of receiving invitations for the moment is only added on the profile page. When this PR is validated, I will make an extra commit to add the same methods in all other pages except game. 

Temporarily and to test the functionality, I enabled the possibility to invite yourself to play.
This will be removed once the PR is validated.

How to test it:

- Login to cosmic pong in an incognito window (to the profile page as it is the only one that manages invitations so far)
- Login to cosmic pong in a normal window and go to chat. Enter a channel and click on the joystick icon next to our own user in order to invite us to play ourselves.

You should receive an invitation in the incognito window.

Test the different possibilities...
